File: app/src/main/java/com/example/android/popularmovies/model/Trailer.java
package com.example.android.popularmovies.model;

import java.util.List;
import javax.annotation.Generated;
import com.google.gson.annotations.SerializedName;

@Generated("com.robohorse.robopojogenerator")
public class Trailer{

	@SerializedName("id")
	private int id;

	@SerializedName("results")
	private List<TrailerItem> results;

	public void setId(int id){
		this.id = id;
	}

	public int getId(){
		return id;
	}

	public void setResults(List<TrailerItem> results){
		this.results = results;
	}

	public List<TrailerItem> getResults(){
		return results;
	}

	@Override
 	public String toString(){
		return 
			"Trailer{" + 
			"id = '" + id + '\'' + 
			",results = '" + results + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/example/android/popularmovies/model/Trailer.kt
package com.example.android.popularmovies.model

import javax.annotation.Generated
import com.google.gson.annotations.SerializedName

@Generated("com.robohorse.robopojogenerator")
class Trailer {

  @SerializedName("id")
  var id: Int = 0

  @SerializedName("results")
  var results: List<TrailerItem>? = null

  override fun toString(): String {
    return "Trailer{id = '$id${'\''.toString()},results = '$results${'\''.toString()}}"
  }
}